third_party/tidy/parser.c

```diff
@@ -4555,9 +4555,9 @@ Node* TY_(ParsePre)( TidyDocImpl* doc, Node *pre, GetTokenMode ARG_UNUSED(mode)
 
                     /*
                       This is basically what Tidy 04 August 2000 did and far more accurate
-                      with respect to browser behaivour than the code commented out above.
+                      with respect to browser behavior than the code commented out above.
```

Comment:
Please keep the British english.

Reply:
Noted, I'm going to fix the spelling to the correct British spelling though. Still a typo